logstash-core/spec/logstash/plugins/registry_spec.rb

```diff
@@ -61,6 +61,13 @@ def register; end
       expect(registry.lookup("filter", "simple_plugin")).to eq(simple_plugin)
     end
 
+    it "should plugin been removed" do
```

Comment:
Can you change this to `it "should be possible to remove the plugin"`

Reply:
Of course.